projects/ngx-datatable/src/lib/themes/material.scss

```
@@ -66,6 +66,10 @@ $datatable-pager-active-background: rgba(158, 158, 158, 0.2) !default;
 $datatable-summary-row-background: #ddd !default;
 $datatable-summary-row-background-hover: #ddd !default;
 
+$datatable-ghost-loader-background-light:  #f5f5f5;
+$datatable-ghost-loader-background:  #d9d8d9;
+$datatable-ghost-loader-background-dark:  #ccc7c7;
```

For consistency's sake, we should also default the values here:

```suggestion
$datatable-ghost-loader-background-light: #f5f5f5 !default;
$datatable-ghost-loader-background: #d9d8d9 !default;
$datatable-ghost-loader-background-dark: #ccc7c7 !default;
```